docrawl/docrawl_core.py

```diff
@@ -974,45 +965,18 @@ def increment_time_of_screenshot_thread(self,screenshot_refreshing_timespan = 5)
                 self.screenshot_thread.stop()
                 self.screenshot_thread.join()
                 self.screenshot_thread = None
-                
 
     def parse(self, response):
-        
-        browser_meta_data=self.docrawl_client.get_browser_meta_data()
-        browser_request = browser_meta_data.get("request")
-        while browser_request is None:
-            
-            
-            # docrawl_logger.info('Waiting for getting a request in browser metadata')
-            time.sleep(1)
-            
-            browser_meta_data=self.docrawl_client.get_browser_meta_data()
-            browser_request = browser_meta_data.get("request")
```

Comment:
Nono, we need to load the website before the loop

Reply:
In my implementation it is unnecessary

Reply:
:/ this will propagate back later...